src/Common/tests/System/PlatformDetection.cs

```diff
@@ -12,6 +12,8 @@ namespace System
 {
     public static class PlatformDetection
     {
+        public static bool IsFullFramework => RuntimeInformation.FrameworkDescription.StartsWith(".NET Framework", StringComparison.OrdinalIgnoreCase);
```

Comment:
Do you feel like replacing the 5 or so other hits on `RuntimeInformation.FrameworkDescription.StartsWith(".NET Framework"